doc/guides/bluetooth/bluetooth-dev.rst

```diff
@@ -82,18 +82,33 @@ combination of two devices:
 Refer to :ref:`bluetooth_qemu_posix` for full instructions on how to build and
 run an application in this setup.
 
-Native POSIX with BabbleSim
-===========================
+Simulated nRF52 with BabbleSim
+==============================
 
 .. note::
    This is currently only available on GNU/Linux
 
-`BabbleSim`_ is a simulator of the physical layer of shared medium networks,
-and it can be used to develop BLE applications with Zephyr when combined with
-the Native POSIX target.
-When developing with BabbleSim, only :ref:`Combined builds
-<bluetooth-build-types>` are possible, since the whole Linux application is a
-single entity that communicates with the simulator.
+The :ref:`nrf52_bsim board <nrf52_bsim>`, is a simulated target board
+which emulates the necessary peripherals of a nrf52 SOC to be able to develop
+and test BLE applications.
+This board, uses:
+
+   * `BabbleSim`_ to simulate the nrf52 modem and the radio environment.
+   * The POSIX arch to emulate the processor.
```

Comment:
Let's use Native POSIX. I've use this name all over

Reply:
Unless Native POSIX is the board and POSIX the arch, in which case could you please clarify that in here?

Reply:
"native_posix" is the board, the "POSIX arch" is the architecture which both "native_posix" and "nrf52_bsim" use :)
If you used native_posix and  posix arch interchangeably we should fix that, otherwise it will add to the confusion.

Reply:
I have used "Native POSIX" all over which makes sense because I was talking about `-DBOARD=native_posix`

Reply:
Let's leave this one as-is.